index.js

```diff
@@ -31,7 +31,9 @@ module.exports = {
   },
 
   importPolyfill: function(app) {
-    app.import('vendor/browser-polyfill.js', { prepend: true });
```

Comment:
This should use `this.import` when present. That would enable this to work properly for nested addons (like ember-prop-types), whereas the guard simply makes it do nothing when used as a nested addon.

Something like:

```js
if (this.import) {  // support for ember-cli >= 2.7
  this.import('vendor/browser-polyfill.js', { prepend: true });
} else if (app.import) { // support ember-cli < 2.7
  app.import('vendor/browser-polyfill.js', { prepend: true });
}
```